build.gradle

```diff
@@ -1,6 +1,54 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.palantir.docker' version '0.21.0'
+    id 'com.palantir.docker-compose' version '0.21.0'
+    id 'com.diffplug.gradle.spotless' version '3.20.0'
+}
+
 group 'multi-int-store'
 version '0.1.0-SNAPSHOT'
 
 repositories {
     mavenCentral()
-}
\ No newline at end of file
+}
+
+allprojects {
+
+    apply plugin: 'com.diffplug.gradle.spotless'
+    apply plugin: 'maven-publish'
+    apply plugin: 'java'
+
+    sourceCompatibility = 1.11
+    targetCompatibility = 1.11
+
+    spotless {
+        java {
+            licenseHeaderFile rootProject.file("license.java")
+            removeUnusedImports()
+            googleJavaFormat()
+            targetExclude '**/GatewayApplication.java'
+        }
+    }
+}
+
+subprojects {
+
+    buildscript {
+        repositories {
+            mavenLocal()
+            mavenCentral()
+        }
+
+        dependencies {
+            classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.1.3.RELEASE'
```

Comment:
✏️ Is this change related to your PR?

Reply:
No but It'll be needed once we start using spring boot. I'll hold off on it